src/Chapter4.hs

```diff
@@ -276,6 +276,13 @@ below. 'Secret' is either an unknown trap with something dangerous
 inside or a reward with some treasure. You never know what's inside
 until opened! But the 'Functor' instance allows changing the reward
 inside, so it is quite handy.
+
+fmap (replicate 3) (Reward "42")
+Reward ["42","42","42"]
+fmap (replicate 3) (Reward 42)
+Reward [42,42,42]
+fmap (replicate 3) (Trap 42)
```

Comment:
If you like, you can even convert these comments into doctest tests :slightly_smiling_face: 

```suggestion
>>> fmap (replicate 3) (Reward "42")
Reward ["42","42","42"]
>>> fmap (replicate 3) (Reward 42)
Reward [42,42,42]
>>> fmap (replicate 3) (Trap 42)
??? what will be here ???
```